tests/integration/cli/query.go

```diff
@@ -157,3 +160,69 @@ func QueryProposal(container, id string) (govtypesv1.Proposal, error) {
 	}
 	return resp, nil
 }
+
+func QueryStakingParams() (stakingtypes.Params, error) {
+	res, err := Query("staking", "params")
+	if err != nil {
+		return stakingtypes.Params{}, err
+	}
+
+	var resp stakingtypes.Params
+	err = helpers.Codec.UnmarshalJSON([]byte(res), &resp)
+	if err != nil {
+		return stakingtypes.Params{}, err
+	}
+
+	return resp, nil
+}
+
+func QueryFeemarketParams() (feemarkettypes.Params, error) {
+	res, err := Query("feemarket", "params")
+	if err != nil {
+		return feemarkettypes.Params{}, err
+	}
+
+	var resp feemarkettypes.Params
+	err = helpers.Codec.UnmarshalJSON([]byte(res), &resp)
+	if err != nil {
+		return feemarkettypes.Params{}, err
+	}
+
+	return resp, nil
+}
+
+func QueryKeysList() (string, error) {
+	res, err := Query1("", "keys", "list", "--keyring-backend", "test")
+	return res, err
+}
+
+func Query1(module, query string, queryArgs ...string) (string, error) {
+	args := []string{"", module, query}
+	args = append(args, queryArgs...)
+	return Exec(args...)
+}
```

Comment:
are we using these anywhere?

Reply:
I'd say we remove bottom 2, while keeping both querying utilities for reference purposes.